README.md

```diff
@@ -432,15 +433,24 @@ I hope I did justice to your time and helped you understand the basics of promet
 **Where to go from here:**
 - It is important to understand PromQL extensively to take advantage of the metrics which one has collected. Remember the goal is not just to collect metrics but to derive answers for application related questions.  [This](https://medium.com/@valyala/promql-tutorial-for-beginners-9ab455142085) is a very good resource to get started with PromQL.
 
-
-
 To Do 
 - [x] Integration with grafana to create dashboards
 - [ ] Add code samples for all metric types.
 - [ ] Explain about the concept of Service Discovery for integrating with kubernetes.
 - [ ] Basic Alerting + Prometheus alerts vs Grafana alerts.
 - [ ] Integrating alerts with tool like pagerduty.
 
+### Run prometheus in docker
+
+You can run example in docker:
```

Comment:
Please be a little more elaborative here what all services will be run using docker also I would suggest adding a link to the docker-compose file.

Reply:
Sure, propose you changes.

Reply:
```suggestion
You can run the whole setup of simple ping server, Prometheus and Grafana using docker. Just run the following command.
```